### DISCUSSION TITLE
Issue with new GitHub Ui Update

### DISCUSSION THREAD

**Discussion by Masterolic** [community/community#55359] [Mobile] [answered]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hello sir/Madam I think you will read this mail/consern as I Found GitHub had rolled out the Beta UI of GitHub.com web interface.Its very good and more tools are available but sad thing not all the GitHub users are having Pc/Desktop as we use mobile phones it's gets lag and can't do space clear and etc... GitHub mobile lack of features and having some issues which I will be attaching image with this email :( I know 70% of GitHub users Have Desktop/Pc please consern about poor phone users :(
Please make optimise for Phone users

Thanks for Replying/Taking to consern list

Thanking you...
by Masterolic 

Device Info:
Running Os: Android 13
Device Ram: 8
Device Cpu core : Octa core,Mediatek
Manufacturer: Vivo 
Model: Y73
![Screenshot_20230511_192136](https://github.com/community/community/assets/93469093/f44b2204-e0fe-4527-87f4-1b8c6908ebbe)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Masterolic ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Masterolic** [reply]
Thank you for the reply queenofcorgis:(
I am sorry if I committed in the wrong topic category but i didn't find any other for GitHub  bug issues if you can guide me thank you :(

Can you show me where you moved it please I'f possible

**Reply by queenofcorgis** [reply, marked as answer]
Hi there - I moved it to our Mobile category, you're already here!

**Reply by Masterolic** [reply]
Thank you @queenofcorgis for website supporting in phone is there any special category
